src/main/webapp/scripts/tests/popular-test.js

```diff
@@ -0,0 +1,5 @@
+QUnit.test('Popular Videos - getPopularRequest', function(assert) {
```

Comment:
How about adding tests for checking that errors are handled correctly? Not too sure how to do this with QUnit though

Reply:
I have that in an upcoming PR where users can select countries, in the current PR the option for 'US' region is hard-coded in so there's no other input options for the getPopularRequest function